src/app/formOverview/getFormCount/getFormCountExpired.lambda.ts

```diff
@@ -37,7 +38,7 @@ export async function handler(event: any) {
     createdBy: 'default_change_to_api_queryparam',
     formName: `ExpiringFormsOverview-${formsRetrievedBeforeDate}-${epochTime}`,
     formTitle: `ExpiringFormsOverview ${formsRetrievedBeforeDate}`,
-    queryStartDate: date.toISOString().substring(0, 10) ?? '',
+    queryStartDate: formsRetrievedBeforeDate,
```

Comment:
Gaat dit goed @WietekeNijmegen ?(regel 12/13 is gewijzigd omdat ik magische cijfertjes in dit soort dingen nooit snap :))

Reply:
Goede toevoeging. Er moet nog meer gerefactored worden vermoed ik. Is allemaal nogal snel gegaan in cowork